pkg/osbuild/systemd_unit_create_stage_test.go

```diff
@@ -0,0 +1,69 @@
+package osbuild
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func createSystemdUnit() SystemdServiceUnit {
+
+	var unit = Unit{
+		Description:         "Create directory and files",
+		ConditionPathExists: []string{"!/etc/myfile"},
```

Comment:
Should there also be a `ConditionPathExists: []string{"!/etc/mydir"}` here?

Reply:
I though of using `/mydir` at first, on reading the systemd.ConditionPathExits [definition](https://www.freedesktop.org/software/systemd/man/latest/systemd.unit.html)  considered using file instead.

Reply:
You are right; you would need support for `ConditionPathIsDirectory`

Reply:
ah! it works with `ConditionPathExists`  unless we have user trying to fool the system by creating a file with same name. I can block this PR until we have that support in osbuild- I will create PR there to add the required support.

Reply:
now have a PR: https://github.com/osbuild/osbuild/pull/1666/ for the support.